### DISCUSSION TITLE
Conventional routing capabilities

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4664] [Q&A]
Hi there,

In react router v6 we use to be able to do dynamic route configuration like below

const router = createBrowserRouter(<DYNAMIC ROUTE CONFIG>);

Is it possible to do it conventional routing?
and also whether we can use hash or memory router for conventional routing?

thanks :)

**Comment by chenjiahan** [comment]
You can disable the `supportHtml5History` option to use HashRouter, ref: https://modernjs.dev/en/configure/app/runtime/router.html#supporthtml5history

> Is it possible to do it conventional routing?

You can't do this with conventional routing, it's controlled by the framework.
